core/src/test/java/com/yahoo/oak/SeqExpandMemoryManagerTest.java

```diff
@@ -22,6 +23,12 @@ public void setUp() {
         seqExpandMemoryManager = new SeqExpandMemoryManager(keysMemoryAllocator);
     }
 
+    @After
+    public void tearDown() {
+        seqExpandMemoryManager.close();
```

Comment:
Why do we need a special `close()` here? And not in the rest of the tests?

Reply:
In the other tests, we close the Oak instance, which closes the MM. Here we don't have Oak instance.